JobToResume.md

```diff
@@ -0,0 +1,69 @@
+Job To Resume
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+JobToResume Recommend resumes based on job data. JobToResume is available at 
```

Comment:
Typo here, and should probably be `The Job To Resume service recommends resumes... The Job To Resume service is available at ... '